CorsixTH/Src/th_gfx_font.cpp

```diff
@@ -702,7 +710,7 @@ text_layout freetype_font::draw_text_wrapped(render_target* pCanvas, const char*
             // Calculate the line height and baseline position.
             FT_Pos iLineHeight = 0;
             FT_Pos iBaselinePos = 0;
-            for(const char* s = itr->first; s != itr->second; )
+            for(const char* s = itr->first; s < itr->second; )
```

Comment:
These changes aren't actually necessary - but I feel more comfortable with them. I got a buffer overflow when I messed up the number of skipped bytes above originally.